### PR TITLE
own: delete `search-ownership` feature flag, making Own enabled by default 🎉

### PR DESCRIPTION
**Pull request by sashaostrikov**
### Run sg locally from this branch if you can, just in case.

Test plan:
Local sg run and tests that Own still works.

Closes https://github.com/sourcegraph/sourcegraph/issues/53554

### CODE REVIEW

File: client/web/src/enterprise/own/RepositoryOwnPage.tsx [hunk 1]
```diff
@@ -68,24 +57,6 @@ export const RepositoryOwnPage: React.FunctionComponent<RepositoryOwnAreaPagePro
         setOpenAddOwnerModal(false)
     }, [])
 
-    useEffect(() => {
-        if (status !== 'initial' && ownEnabled) {
-            telemetryService.log('repoPage:ownershipPage:viewed')
```
Comment (cbart) [inline client/web/src/enterprise/own/RepositoryOwnPage.tsx:73]: I think we should keep this event - it gives us data on whether we display drilldown page. We could include it in admin stats.

**Review by cbart — APPROVED** [review]

**Comment by cbart** [comment]
I've tested manually on this branch and everything seems in order :shipit:

File: client/web/src/enterprise/own/RepositoryOwnPage.tsx (hunk 1 above)
Comment (sashaostrikov) [reply client/web/src/enterprise/own/RepositoryOwnPage.tsx:73]: whoops, that was an accident. rage-deleting the lines went sideways

**Review by eseliger — APPROVED** [review]
woo, way to go team!!

**Comment by limitedmage** [comment]
🎉🎉🎉

**Pull request merged**